crates/goose-mcp/src/developer/developer_prompt.rs

```diff
@@ -0,0 +1,25 @@
+use mcp_core::prompt::{Prompt, PromptArgument, PromptTemplate};
+use std::fs;
+use std::path::Path;
+
+pub fn create_unit_test_prompt() -> Prompt {
+    let prompt_path =
+        Path::new(env!("CARGO_MANIFEST_DIR")).join("src/developer/prompts/unit_test.json");
```

Comment:
i don't this will work when we distribute it. you'll need sth like this - https://github.com/block/goose/blob/bcd991c32262e6061d8aaa0989f28958c5c0c764/crates/goose/src/prompt_template.rs#L7